### DISCUSSION TITLE
Latest RAVE VST3/AU/Standalone (23 Aug 2022) not working on MacOs 11.6.5

### DISCUSSION THREAD

**Discussion by KhyamAllami** [acids-ircam/RAVE#120] [Q&A]
Hi everyone,
I'm having trouble getting the latest RAVE VST3/AU or Standalone app to work on MacOs 11.6.5:
https://github.com/acids-ircam/rave_vst/actions/runs/2911472905 (23 Aug 2022)

When trying to launch the AU/VST3 in Ableton Live 11 I get the silly security error but even on choosing "open anyway" it doesn't seem to work. Have tried restarting etc... with no luck.

When trying to run the standalone app I get the following from the error report:

```
Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  dyld: Using shared cache: DC9219E8-D54C-369D-81BB-CF54BFB2F6CD
Library not loaded: @rpath/libtorch.dylib
  Referenced from: /private/var/folders/*/RAVE.app/Contents/MacOS/RAVE
  Reason: image not found

```

Any ideas? 
I'm unfortunately unable to make my own build. If anyone has a previous build that is working could you please share?

Thanks a lot,
Khyam

**Comment by justusstellar** [comment]
Same  here cannot open RAVE.

**Reply by justusstellar** [reply]
Also!! if anyone can teach me aspects of how to use this i'd love to buy you a coffee or chat on discord. I'm new, but really want to know a good starting point. Thank you
